lib/facter/nginx_version.rb

```diff
@@ -1,6 +1,6 @@
 Facter.add(:nginx_version) do
   setcode do
-    if Facter.value('kernel') != 'windows' && (Facter::Util::Resolution.which('nginx') || Facter::Util::Resolution.which('openresty'))
+    if Facter.value('kernel') != 'windows' && Facter.value('operatingsystem') != 'nexus' && (Facter::Util::Resolution.which('nginx') || Facter::Util::Resolution.which('openresty'))
```

Comment:
Can we move this to ```confine```? According to https://puppet.com/docs/facter/3.9/fact_overview.html#main-components-of-simple-resolutions you can repeat it multiple times.